grammar/lexer.l

```diff
@@ -100,9 +102,10 @@ extern int yydebug;
 /* somehow, I need these prototype even though the headers are 
  * included. I guess that's some autotools magic I don't understand...
  */
-#if !defined(__FreeBSD__)
+/*#if !defined(__FreeBSD__)
```

Comment:
Aren't you disabling this code block for all platforms? Is this problem fixed? If we really don't need this hack anymore, please get rid of the complete block.